etc/tlspool.cygwin.conf

```diff
@@ -0,0 +1,339 @@
+# tlspool.conf - Configuration of the TLS Pool
+#
+# Lines in this file may only hold one of the following forms:
+#  - nothing: ignored
+#  - only whitespace: ignored
+#  - a # sign and arbitrary characters: ignored
+#  - a word, a single space and arbitrary characters: a config declaration
+#
+# The order of declarations matter; if options are undeclared or appear out
+# of their expected order they may lead to a syntax error.
+#
+
+#
+# The TLS Pool is a daemon.  Set its PID file here, to be used in scripts.
+#
+daemon_pidfile /var/run/tlspool.pid
+
+#
+# The TLS Pool listens to a UNIX domain socket, reachable to users and groups
+# with the proper permissions.  Use mode 0660 to permit same-user and
+# same-group access to the TLS Pool.
+#
+# Note that initial root privileges are needed to be able to change to
+# another socket_user and possibly to change the socket_group.  These
+# actions are performed before dropping privileges.
+#
+# Multiple of these may be specified.  They are instantiated at the point
+# of the socket_name declaration.  This serves the purpose of a simple
+# 
+#
+socket_user hfman
+socket_group hfman
+socket_mode 0666
+socket_name /var/run/tlspool.sock
+
+#
+# The TLS Pool usually drops privileges to a lower-ordered user, as
+# specified below.  Note that the PKCS #11 interface is a library,
+# so it may suffer from a chroot if it operates on locally stored
+# data that is not available under the chroot environment.
+#
+# Note that chroot must be performed before changing user/group.
+#
+# daemon_chroot /var/chroot/tlspool
+daemon_user hfman
+daemon_group hfman
```

Comment:
Dit werkt onder Cygwin:
  ...
  # When using Cygwin use current user and group
  #socket_user tlspool
  #socket_group tlspool
  ...
  # When using Cygwin use current user and group
  #daemon_user tlspool
  #daemon_group tlspool